### PR TITLE
known_shell_spawn_cmdlines - lighttpd

### PR DESCRIPTION
**Pull request by mmonitz**
Signed-off-by: Matan Monitz <mmonitz@gmail.com>

**What type of PR is this?**

/kind bug
/kind rule-update

/area rules

**What this PR does / why we need it**:
lighttpd is spawning create-mime.conf.pl on startup, causing a false positive for "Run Shell Untrusted"
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
lighttpd is spawning create-mime.conf.pl on startup, causing a false positive for "Run Shell Untrusted"
add `sh -c /usr/share/lighttpd/create-mime.conf.pl` to `known_shell_spawn_cmdlines` macro
```

### CODE REVIEW

**Comment by gstrauss** [comment]
FYI: lighttpd provides create-mime.conf.pl as an option to generate lighttpd mime configuration from /etc/mime.types.  The lighttpd official source tree ships with a pre-generated doc/config/conf.d/mime.conf which can be used instead of create-mime.conf.pl, but not all packagers make that available.  Another option is that you can also define your own custom `mimetype.assign` config in lighttpd.conf.

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -2065,6 +2065,7 @@
     '"sh -c openssl version"',
     '"bash -c id -Gn kafadmin"',
     '"sh -c /bin/sh -c ''date +%%s''"'
+    '"sh -c /usr/share/lighttpd/create-mime.conf.pl"'
```
Comment (leogr) [inline rules/falco_rules.yaml:2068]:
```suggestion
    '"sh -c /bin/sh -c ''date +%%s''"',
    '"sh -c /usr/share/lighttpd/create-mime.conf.pl"'
```

Just a missing comma. Indeed, tests are failing for that :point_down: 
```
Could not load YAML file: yaml-cpp: error at line 2068, column 5: end of sequence flow not found
```

**Review by leogr — COMMENTED** [review]
Hey @mmonitz 

Thank you for your contribution! Your PR SGTM, but I have found an issue. See my suggestion.

File: rules/falco_rules.yaml (hunk 1 above)
Comment (mmonitz) [reply rules/falco_rules.yaml]: whoopsie, sorry about that!

**Review by leogr — APPROVED** [review]
LGTM :+1: 

Thank you!

/approve

**Review by Kaizhe — APPROVED** [review]

**Comment by jasondellaluce** [comment]
Closing and reopening to trigger @poiana 😃 

/close

**Pull request merged**